src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vm/_reimage.py

```diff
@@ -53,7 +53,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
         )
         _args_schema.vm_name = AAZStrArg(
             options=["-n", "--name", "--vm-name"],
-            help="The name of the virtual machine.",
+            help="The name of the Virtual Machine. You can configure the default using `az configure --defaults vm=<name>`",
```

Comment:
Is this a new feature automatically supported by Code Gen?

Reply:
It is a new feature of Code Gen V2, which supports setting default values by `az configure`